release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2477 @@
+
+{product-title} 3.6 introduces improved router customization documentation. Many
+RFEs could be solved with better documentation around the HAProxy
+features and functions which are now added, and their customizable fields via
+annotations and environment variables. For example, router annotations to do
+per-route operations.
+
+For example, to change the behavior of HAProxy (round-robin load balancing)
+through annotating a route:
+
+----
+$ oc annotate route/ab haproxy.router.openshift.io/balance=roundrobin
+----
+
+For more information, see
+xref:../install_config/router/customized_haproxy_router.adoc#install-config-router-customized-haproxy[Deploying
+a Customized HAProxy Router].
+
+[[use-a-different-f5-partition]]
+==== Use a Different F5 Partition Other than /Common
+
+With {product-title} 3.6, there is now the added ability to use custom F5
+partitions for properly securing and isolating {product-title} route
+synchronization and configuration.
+
+The default is still `/Common` or global partition if not specified. Also,
+behavior is unchanged if the partition path is not specified.  This new feature
+ensures all the referenced objects are in the same partition, including virtual
+servers (`http` or `https`).
+
+[[ocp-36-support-ipv6-terminated-at-the-router-with-internal-ipv4]]
+==== Support IPv6 Terminated at the Router with Internal IPv4
+
+The router container is able to terminate IPv6 traffic and pass HTTP[S] through
+to the back-end pod.
+
+The IPv6 interfaces on the router must be enabled, with IPv6 addresses listening
+(`::80`, `::443`). The client needs to reach the router node using IPv6.
+IPv4 should be unaffected and continue to work, even if IPv6 is disabled.
+
+[NOTE]
+====
+HAProxy can only terminate IPv6 traffic when the router uses the network stack
+of the host (default). When using the container network stack (`oadm router
+--service-account=router --host-network=false`), there is no global IPv6 address
+for the pod.
+====
+
+[[ocp-36-installation]]
+=== Installation
+
+[[ocp-36-automated-installtion-of-cloudforms]]
+==== Automated installation of CloudForms 4.5 Inside OpenShift (Technology Preview)
+
+The installation of containerized CloudForms inside {product-title} is now part
+of the main installer (currently in xref:ocp-36-technology-preview[Technology
+Preview]). It is now treated like other common components (metrics, logging, and
+so on).
+
+After the {product-title} cluster is provisioned, there is an additional
+playbook you can run to deploy CloudForms into the environment (using the
+`openshift_cfme_install_app flag` in the hosts file).
+
+----
+$ ansible-playbook -v -i <INVENTORY_FILE> playbooks/byo/openshift-cfme/config.yml
+----
+
+Requirements:
+
+[cols="4*", options="header"]
+|===
+|Type
+|Size
+|CPUs
+|Memory
+
+|Masters
+|1+
+|8
+|12 GB
+
+|Nodes
+|2+
+|4
+|8 GB
+
+|PV Storage
+|25 GB
+|N/A
+|N/A
+|===
+
+[NOTE]
+====
+NFS is the only storage option for the Postgres database at this time.
```

Comment:
It's better to also mention that 1) NFS server should be on the first master host 2)PV backing NFS storage volume is mounted on /exports/